src/mozanalysis/experiment.py

```diff
@@ -254,21 +254,26 @@ def get_time_series_data(
             self.num_dates_enrollment
         )
 
-        sql_template = self.build_query_template(
-            metric_list=metric_list,
+        sql_template_enrollments = self.build_enrollment_query(
             time_limits=time_limits,
             enrollments_query_type=enrollments_query_type,
             custom_enrollments_query=custom_enrollments_query,
             segment_list=segment_list,
         )
 
+        enrollments_table = bq_context.run(sql_template_enrollments)
+
+        sql_template_metrics = self.build_metrics_query(
```

Comment:
This is no longer a template, right?

Reply:
Yeah, the naming here needs some adjustment